lighthouse-core/report/v2/renderer/details-renderer.js

```diff
@@ -90,6 +92,27 @@ class DetailsRenderer {
   }
 
   /**
+   * @param {!DetailsRenderer.LinkDetailsJSON} details
+   * @return {!Element}
+   */
+  _renderLink(details) {
+    const allowedProtocols = ['https:', 'http:'];
```

Comment:
only allows `http` and `https` links, like [`dom. convertMarkdownLinkSnippets `](https://github.com/GoogleChrome/lighthouse/blob/8c3c3b50ae9bf2d83dcc797d0e5063768f6157aa/lighthouse-core/report/v2/renderer/dom.js#L98) does already.

If we need more supported protocols we can add them in the future, but better to err on the side of caution as these links will be coming from things like the hard-to-inspect snyk blob in #2372